deformable_manipulation_experiment_params/launch/cloth_single_pole_params.launch

```diff
@@ -85,11 +89,12 @@
 
     <group ns="smmap_planner_node">
         <group ns="task">
-            <param name="max_time"                              value="$(arg task_max_time)"    type="double"/>
-            <param name="error_threshold_along_normal"          value="0.02"                    type="double"/>
-            <param name="error_threshold_distance_to_normal"    value="0.006"                   type="double"/>
-            <param name="error_threshold_task_done"             value="0.1"                     type="double"/>
-            <param name="max_stretch_factor"            value="$(eval (0.012 + 0.002) / 0.012)" type="double"/>
+            <param name="max_time"                            value="$(arg task_max_time)"            type="double"/>
```

Comment:
The spacing here is to keep the "desired_down_scale" line short. Whether we should keep "desired_down_scale" is another question. The value is set to one here. The resulting object desired motion = task-calculated desired motion / down scale